crates/env/src/engine/off_chain/mod.rs

```diff
@@ -54,7 +54,10 @@ use self::{
     },
 };
 use super::OnInstance;
```

Comment:
Off-chain environment impls, dispatches between off-chain and on-chain or even mentions between the two do not make sense in any environment implementation. This is just important in the `ink_env` crate that combines all available environments.